package.json

```diff
@@ -16,6 +16,7 @@
     "body-parser": "*",
     "codacy-coverage": "^3.4.0",
     "express": "^4.14.0",
+    "global": "^4.4.0",
```

Comment:
@venkateshwarans @pallakartheekreddy  We have added this NPM as a dependency, please look into this